### ISSUE TITLE
got loss like a valley， ‘U’ type， and acc only 20%

### ISSUE THREAD

**Issue by ykk648** [tiandunx/SV-X-Softmax#3]
different learning rate like 1e-1 1e-2 1e-3,
using my own train&val dataset

**Comment by ykk648** [comment]
my val code problem, solved
